projects/plugins/boost/app/data-sync/read-only/Available_Modules.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Data_Sync\Read_Only;
+
+use Automattic\Jetpack_Boost\Modules\Modules;
+
+class Available_Modules extends Storage {
+	public function get( $_key ) {
+		$modules = new Modules();
+		$keys    = array_keys( $modules->available_modules() );
+
+		return array_map(
+			function ( $key ) {
+				return str_replace( '-', '_', $key );
+			},
+			$keys
+		);
```

Comment:
Hey! Quick thing I discovered just now by accident. [`str_replace`'s](https://www.php.net/str_replace) third argument also supports an array, so it looks like you don't need the `array_map`!

So you can do just:
```php
return str_replace( '-', '_', $keys );
```

Reply:
Wow! that is an excellent find.